contracts/Ovm_SpokePool.sol

```diff
@@ -59,6 +59,7 @@ contract Ovm_SpokePool is SpokePool {
         l1Gas = 5_000_000;
         __SpokePool_init(_initialDepositId, _crossDomainAdmin, _hubPool, _wrappedNativeToken, _timerAddress);
         messenger = Lib_PredeployAddresses.L2_CROSS_DOMAIN_MESSENGER;
+        require(_l2Eth != address(0), "Invalid L2 ETH address");
```

Comment:
Question: Slither recommends we add these zero address checks. It does add a fixed gas cost to construction but adds some safety. Thoughts?

Reply:
I'm not sure about the value of these checks. 0 is only one possible invalid address, but there are ~infinitely many other invalid addresses as well. The only way we can really be confident that the address is set correctly is via post-deployment verification.

I'm not firmly against it or anything, it just seems a bit redundant and might give a false sense of security.

Reply:
i think I agree with your comment and will remove these zero address checks